apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -30,19 +30,27 @@
 /****************************************************************************
  * Global Variables
  ****************************************************************************/
-static const char *file_path = MEDIA_TEST_FILE_PATH;
-static const char *test_data = "dummydata";
-static int g_flag=0;
+static const char file_path[] = "/mnt/fileinputdatasource.raw";
+static const char test_data[] = "dummydata";
 static unsigned char *buf;
+static int g_flag=0;
 
 static void SetUp(void)
 {
-	FILE *fp;
-	fp = fopen(file_path, "w");
-	TC_ASSERT_NEQ("fopen", fp, NULL);
-	TC_ASSERT_NEQ_CLEANUP("fputs", fputs(test_data, fp), EOF, fclose(fp));
-	TC_ASSERT_EQ("fclose", fclose(fp), OK);
-	buf = new unsigned char[31];
+	buf = new unsigned char[21];
```

Comment:
Dont hardcode, use #defines if possible.

Reply:
changes done and code also updated in gerrit